README.md

```diff
@@ -216,8 +141,8 @@ This is an open source project open to anyone. This project welcomes contributio
 
 ## Feedback & Questions
 
-If you discover an issue please file a bug in [GitHub issues](https://github.com/redhat-actions/buildah/issues) and we will fix it as soon as possible.
+If you discover an issue please file a bug in [GitHub issues](https://github.com/redhat-actions/buildah-build/issues) and we will fix it as soon as possible.
```

Comment:
Honestly this section and the License section could just be removed.

Reply:
And the Contributing section.

Reply:
We can, as this is well known thing in open source projects.